src/containers/app.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import BooksList from './booksList';
+import store from '../reducers/index';
+import BooksForm from '../components/booksForm';
+
+const App = () => (
+  <Provider store={store}>
+    <BooksList />
+    <BooksForm />
+  </Provider>
+);
+
+export default App;
```

Comment:
Let's move `App` component to the `src/components` directory since `App` is a  presentational component.

![image](https://user-images.githubusercontent.com/36057474/73388483-9ba3f380-42d2-11ea-9b36-3fe49405999a.png)


See https://www.reactnative.guide/9-redux/9.2-presentational-vs-containers.html